csrc/cpu/dnnl_helper.hpp

```diff
@@ -32,6 +33,11 @@ struct DNNLType<c10::BFloat16> {
   static constexpr dnnl::memory::data_type type = dnnl::memory::data_type::bf16;
 };
 
+template <>
+struct DNNLType<c10::Half> {
+  static constexpr dnnl::memory::data_type type = dnnl::memory::data_type::f16;
+};
+
```

Comment:
Seems that DNNL supports f16 as well, could we update the IPEX quantization support dtype to use AWQ quantization with fp16?
https://github.com/vllm-project/vllm/blob/ad23318928d40ef7ac969451afa0dc198428c04b/vllm/model_executor/layers/quantization/ipex_quant.py#L55-L58

Reply:
Updated.